src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/ConfigurationTests.cs

```diff
@@ -147,6 +167,149 @@ public void ConfigurationOrderingTest(ConfigurationLevel level)
             }
         }
 
+        /// <summary>
+        /// Instead of having to explicitly define every expected value, this reuses the individual categories to ensure they
+        /// assemble properly when combined.
+        /// </summary>
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
+        public void FullConfigurationTest(bool redact)
+        {
+            using TemporaryDirectory userSettingsDirectory = new(_outputHelper);
```

Comment:
Rather than doing this, could you just pass the content that you want written as the settings.json file directly tho the ConfigShowOutputTest method? I don't see any purpose for writing it to disk first; maybe this is a relic of earlier iterations.

Reply:
Yep, good catch; I did a fair amount of trimming and reorganizing yesterday to improve maintainability and didn't even think twice about the duplication.